website_updater.py

```diff
@@ -95,21 +110,44 @@ def request_responses(data_url: str) -> List[Dict[str, str]]:
     return responses
 
 
-def cleanup_repo(repo: str, branch_name: str):
-    execute_cmd(["git", "restore", "."], repo)
-    execute_cmd(["git", "checkout", "master"], repo)
-    execute_cmd(["git", "reset", "--hard", "HEAD"], repo)
-    execute_cmd(["git", "pull"], repo)
-    execute_cmd(["git", "checkout", "-b", branch_name], repo)
+def cleanup_repo(repo: str, branch_name: str, dry_run: bool):
+    execute_cmd(["git", "restore", "."], repo, dry_run)
+    execute_cmd(["git", "checkout", "master"], repo, dry_run)
+    execute_cmd(["git", "reset", "--hard", "HEAD"], repo, dry_run)
+    execute_cmd(["git", "pull"], repo, dry_run)
+    execute_cmd(["git", "checkout", "-b", branch_name], repo, dry_run)
```

Comment:
I think `execute_cmd` and it's callers would look nicely together as methods of some class keeping `dry_run` et al. in its state. But given the nature of this script it's probably not worth the effort.